webapp/lib/controller.dart

```diff
@@ -117,6 +195,13 @@ void command(UIAction action, Data actionData) {
     /*** Data */
     case UIAction.projectSelected:
       view.contentView.populateUrlFilters();
+
+      var selectedProjectTimer = projectTimers[view.contentView.projectSelectorView.selectedProject];
+      if (selectedProjectTimer != null) {
```

Comment:
I believe the condition should be `selectedProjectTimer != null && selectedProjectTimer.isActive == true)`?

Reply:
Yes. Fixed